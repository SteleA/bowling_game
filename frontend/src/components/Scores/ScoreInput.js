import React from 'react';
import { FormGroup, FormControl } from 'react-bootstrap';

export default props => (
  <FormGroup>
    <FormControl
      id={props.id}
      type="number"
      value={props.input}
      onChange={({ target: { value } }) => props.onChange(value)}
      placeholder={props.placeholder}
    />
  </FormGroup>
);
