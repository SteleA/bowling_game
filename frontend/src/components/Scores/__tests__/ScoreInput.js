import { shallow } from 'enzyme';  // eslint-disable-line
import { shallowToJson } from 'enzyme-to-json';  // eslint-disable-line
import React from 'react';
import ScoreInput from '../ScoreInput';

const props = {};

describe('<ScoreInput />', () => {
  it('should render with default props', () => {
    const wrapper = shallow(
      <ScoreInput {...props} />
    );
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
