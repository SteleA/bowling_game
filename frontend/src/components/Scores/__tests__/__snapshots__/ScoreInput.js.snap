// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ScoreInput /> should render with default props 1`] = `
<FormGroup
  bsClass="form-group"
>
  <FormControl
    bsClass="form-control"
    componentClass="input"
    onChange={[Function]}
    type="number"
  />
</FormGroup>
`;
